neurom/io/check.py

```diff
@@ -79,7 +77,7 @@ def has_increasing_ids(raw_data):
     with their predecessor)
     '''
     ids = raw_data.get_col(COLS.ID)
-    steps = [int(j) for (i, j) in zip(ids, ids[1:]) if i >= j]
+    steps = ids[np.where(np.diff(ids) <= 0)[0] + 1].astype(int)
```

Comment:
Well, I was returning ints before, and they are supposed to be IDs. That is the only reason I did the conversion. I may look at using one of those mixed type numpy arrays, but I am worried it will slow things down.